src/components/common/inputs/file-input.less

```diff
@@ -50,4 +50,13 @@
     visibility: hidden;
     position: absolute;
   }
+
+  .button-container {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
Originally this project supported css styles for old Internet Explorer browser. Now that Microsoft announced that they no longer going to maintain EI browser at all, I'm so happy to finally use some new css features like `display: flex` 😃